exercises/tournament/tournament_test.go

```diff
@@ -127,11 +127,10 @@ func TestTallyError(t *testing.T) {
 	for _, s := range errorTestCases {
 		reader := strings.NewReader(s)
 		var buffer bytes.Buffer
-		err := Tally(reader, &buffer)
+		var err error = Tally(reader, &buffer)
```

Comment:
For consistency, perhaps make same change in func TestTallyHappy.

Reply:
Thanks for reviewing! The one in TestTallyHappy might not be necessary as returning non-error type will fail the test anyway.